examples/tests/cloud-push-pull/Earthfile

```diff
@@ -0,0 +1,136 @@
+FROM ../../..+earthly-docker
+WORKDIR /test
+
+RUN apk update && \
+    apk add curl python3 bash
+
+all:
+    BUILD +google-artifact-registry
+    BUILD +google-container-repository
+    BUILD +azure-container-registry
+    BUILD +amazon-elastic-container-registry
+
+google-base:    
+    RUN curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-324.0.0-linux-x86_64.tar.gz && \
+        tar -xvzf google-cloud-sdk-324.0.0-linux-x86_64.tar.gz && \
+        ./google-cloud-sdk/install.sh -q && \
+        ls -la /test/google-cloud-sdk
+    
+    ENV PATH $PATH:/test/google-cloud-sdk/bin
+
+    RUN --secret=GCP_KEY=+secrets/earthly-technologies/gcp/ci-cd-key \
+        echo $GCP_KEY > key.json
+
+    RUN gcloud auth activate-service-account --key-file /test/key.json
```

Comment:
The GCP is under my name; if you need or want access yell? Vlad might defacto have access?